src/DGtal/images/ImageContainerByHashTree.h

```diff
@@ -276,6 +276,19 @@ namespace DGtal
      */
     Range range() ;
 
+    /**
+     * Give access to the underlying container.
+     * @return a (might be const) reference to the container.
+    */
+    inline const Self & container() const { return *this; };
+    inline Self & container() { return *this; };
+
+    /**
+     * Give access to the underlying data.
+     * @return a (might be const) reference to the data.
+    */
+    inline const Node** & data() const noexcept { return myData; };
+    inline Node** & data() noexcept { return myData; };
```

Comment:
Same question here and at everyplace afterwards.

Reply:
See other comment in the same topic